.swiftlint.yml

```diff
@@ -16,6 +16,10 @@ line_length:
 type_body_length:
   # error: 350
   error: 500
+  
+function_body_length:
+  error: 200
```

Comment:
Yes, I think function with the maximum length is here table view cellForRowAt, because of switch-case for sections... I can revert the rule. 
I have changed the rule, because I got an error, not a warning, when I wanted to add new line in method.

Reply:
I see.. that would be a good candidate for refactoring. The easy solution would be to make functions for each section:
```swift
func userSettingsCell(for indexPath: IndexPath) -> UITableViewCell {
```
But maybe you can find an even better solution e.g. by using protocols for abstraction.

Also, remember that Swift supports type extensions. That can be really helpful in splitting a big ViewController up into smaller parts or even files (e.g. keep only the view life cycle in the original class and use extensions for: UITableView Delegate/DataSource, user interaction, app logic, etc.)

Reply:
Yes I put it in extension of class. Method to setup every section cell.

Reply:
That's great, thanks.
Do you still need the rule change then?